ember-learn/ember-blog
# The Ember Times No. 154 - June 26th 2020

📝  Ideas, feel free to add to list or claim!

- Readers' Questions
  - [ ] Preferred ways to replace mixins in Ember Octane
  - [x] How to become a writer for The Ember Times (🔒 @ijlee2)
  - [ ] When to start using Embroider for production apps
- [ ] A new release of `ember-place-autocomplete`
  - https://twitter.com/dmuneras/status/1265530766739603457
- [ ] v1.2 release of `ember-concurrency` and v2.0 of `ember-concurrency-decorators`
  - https://twitter.com/m4xm4n/status/1273404988845367297
- [ ] New addon: `ember-validation-state`
  - https://github.com/chrismllr/ember-validation-state
  - > Just released an addon which provides a decorator for form validation, on top of ember-validators:
     >
     > Utilizes patterns defined in ember-cp-validations, and brings in some of the intl support a la ember-intl/cp-validations
- [ ] welcome isaac and jared https://twitter.com/emberjs/status/1273942812346720256
- [x] New addon: `ember-stargate` (🔒 @chrisrng )
  - https://twitter.com/simonihmig/status/1274066847873273859
- [ ] New addon: `ember-custom-elements`
  - https://github.com/Ravenstine/ember-custom-elements
- [ ] New addon: `empress-blog-netlify-cms`(:lock: @jaredgalanis)
  - https://github.com/MalayaliRobz/empress-blog-netlify-cms
- [x] Part 3 of Abhilash's blog series on app optimization
  - https://abhilashlr.in/ember-performance-tweaks-part-3 (:lock_with_ink_pen: @jayjayjpg)
- [ ] Ember 101. A new Ember tutorial series by Aamir Mahmood
  - https://www.youtube.com/playlist?list=PLo3Uqgz3Y_QIS5cDRZdynGF3B8i1lgsOs
  - Whoever takes this, please watch and review the videos for quality.
- [ ] Rock and Roll with Ember.js Octane edition will be released on 6/25
  - https://twitter.com/baaz/status/1276104785079472130
  - Ask from Learning Team - coordinate news with Balint on the writeup
- [ ] Tweaking UI Behavior with Ember Data Queries https://codingitwrong.com/2020/06/23/ember-list.html

📆  Future stories

- For the next issue after 6/25 add follow up coverage to the a11y team’s blog post
- Also a heads up, the roadmap RFC was merged but a blog post is upcoming and that's what should be publicised (from @locks - follow this PR https://github.com/ember-learn/ember-blog/pull/646)
- Deprecate string based actions RFC https://github.com/emberjs/rfcs/pull/632, check on messaging with locks 
- Updates on Yehuda Octane series (when new posts after April 6 one are out): https://yehudakatz.com/2020/04/06/ember-octane-airtable-time/
- Updates on pzuraq autotracking series https://www.pzuraq.com/autotracking-case-study-trackedmap/

🤔  Questions?

See #support-ember-times on Discord or CONTRIBUTING.

✅  Todos:

- Check https://github.com/ember-learn/whats-new-in-emberland (roundup of all the PRs in emberjs and ember-learn repos) for ideas
- Use https://github.com/ember-learn/emberjs-times-tools README
- Check Netlify preview, to ensure display on emberjs.com/blog is good
  - [ ] Check buttons
- [ ] Crosspost on dev.to, delete Readers Question picture, add Canonical URL 
- [ ] Set up tweet for Monday morning on tweetdeck
  - [ ] post $natural
